queries.sql

```diff
@@ -7,4 +7,39 @@ SELECT date_of_birth FROM animals WHERE name ='piachu' OR name ='Agumon';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+
+BEGIN;
+  update animals
+  set species = 'unspecified';
+  ROLLBACK;
+
+  BEGIN;
+  update animals
+  set species = 'digimon' where name like '%mon';
+  update animals
+  set species = 'pokemon' where name not like '%mon';
```

Comment:
* Kindly verify that the change was made  here as requested in the[ project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.)